packages/react-composites/tests/browser/call/MockCallAdapterState.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  CallState,
+  LocalVideoStreamState,
+  RemoteParticipantState,
+  RemoteVideoStreamState,
+  VideoStreamRendererViewState
+} from '@internal/calling-stateful-client';
+import type { CallAdapterState } from '../../../src';
+
+/**
+ * A slight modification of {@link CallAdapterState} for initializing the
+ * {@link MockCallAdapter} for hermetic e2e tests.
+ *
+ * - HTMLElement views are replaced with placeholders to be populated
+ *   by the test app.
+ */
```

Comment:
Thanks for creating these Mock interfaces. Now it is tightly coupled with the state interfaces of in 'calling-stateful-client'